ironfish/src/consensus/verifier.ts

```diff
@@ -86,7 +90,7 @@ export class Verifier {
     // minersFee should be (negative) miningReward + totalTransactionFees
     const miningReward = block.header.strategy.miningReward(block.header.sequence)
     if (minersFee !== BigInt(-1) * (BigInt(miningReward) + totalTransactionFees)) {
-      return { valid: false, reason: VerificationResultReason.INVALID_MINERS_FEE }
+      return { valid: false, reason: VerificationResultReason.INVALID_FEE_SUM }
```

Comment:
I think this could stay as INVALID_MINERS_FEE, since it's making sure that the minersFee derived field on the block header is actually correct.

Reply:
Agree. The other 2 INVALID_MINERS_FEE I will change so we don't have different failure modes colliding with the same error code and to ensure our tests actually generate the failures we're expecting.